lib/msf/core/exploit/remote/tcp_server.rb

```diff
@@ -54,12 +54,11 @@ def on_client_close(client)
   #
   # Starts the service.
   #
-  def start_service(*args)
+  def start_service(opts = {})
```

Comment:
Update the `Exploit::Remote::SocketServer#start_service` signature to reflect this as well.